src/ConnectionRequest.jl

```diff
@@ -39,9 +37,6 @@ function request(::Type{ConnectionPoolLayer{Next}}, url::URI, req, body;
     end
 end
 
-
-sockettype(url::URI, default) = url.scheme in ("wss", "https") ? SSLContext :
-                                                                 default
-
+sockettype(url::URI, default) = url.scheme in ("wss", "https") ? SSLContext : default
```

Comment:
This might seem like weird formatting on a wide terminal, but I have taken some care to keep lines of code less than 80 columns where possible.